core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -384,6 +384,13 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
                 enableSslNative = properties.getProperty("quarkus.ssl.native") != null
                         ? Boolean.parseBoolean(properties.getProperty("quarkus.ssl.native"))
                         : false;
+
+                if (!addAllCharsets) {
+                    final String allCharsetsProperty = properties.getProperty("quarkus.add-all-charsets");
```

Comment:
Sorry for the late comment but maybe we should rename this one to `quarkus.native.enable-all-charsets`.